### ISSUE TITLE
Feature Request: Ability to configure different indenting on template literals

### ISSUE THREAD

**Issue by luislobo** [eslint/eslint#10942]
**The version of ESLint you are using.**
5.4.0
**The problem you want to solve.**
Ability to configure `indent` rule for template literals

**Your take on the correct solution to problem.**

Right now, the following code:
```javascript
const example = `This is a nice template literal ${1 + 1
  }. Like it?`;

module.exports = example;
```
throws this error:
```
/home/luis/tmp/index.js
  2:1  error  Expected indentation of 0 spaces but found 2  indent
```
I would like to be able to specify, for text literals, that I want to indent it to 2 (or any other configurable number of spaces) after the previous line.

I didn't find a way to set in the documentation.

**Comment by izanf** [comment]
You have two spaces between } on second line.

**Comment by luislobo** [comment]
@izanf Yes, that is intentional! that is what I want to be able to do... I'm not asking for "what is the error here". I just want to be able to configure eslint to allow for two spaces after the second line.

**Comment by kaicataldo** [comment]
I wonder if this is actually a bug. None of the following examples produce any errors (the rule seems to ignore the indentation when the line starts with another token that's part of the template literal expression):

```
const example = `This is a nice template literal ${1 
  + 1}. Like it?`;

module.exports = example;
```
```
const example = `This is a nice template literal ${1 
+ 1}. Like it?`;

module.exports = example;
```
```
const example = `This is a nice template literal ${1 
+ 
   1}. Like it?`;
```

Given the above isn't enforce, it seems to me like the case where the closing brace is at the beginning of the line should be treated similarly.

**Comment by not-an-aardvark** [comment]
@kaicataldo I think the behavior you're observing is a result of the `indent` rule never enforcing any indentation for the operator/RHS of binary expressions. It's not related to template literals in particular.

**Comment by luislobo** [comment]
@not-an-aardvark well, this issue is for basically adding the ability to configure those cases in template literals, in case this is some "hardcoded" rule, or if it's a bug, to fix it...

**Comment by kaicataldo** [comment]
@not-an-aardvark Thanks for clarifying that!
